Parse/PFObject.h

```diff
@@ -569,7 +569,7 @@ NS_REQUIRES_PROPERTY_DEFINITIONS
 
  @returns The task that encapsulates the work being done.
  */
-- (BFTask *)fetchInBackground;
+- (BFTask PF_GENERIC(__kindof PFObject *)*)fetchInBackground;
```

Comment:
Can we use `instancetype`? It's ok if not - I am just wondering if it will let us do that, since this is purely `BFTask` incapsulating `self`.